src/Microsoft.ML.Fairlearn/reductions/Utilities.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.ML.Data;
+using Microsoft.ML.SearchSpace;
+using Microsoft.ML.SearchSpace.Option;
+
+namespace Microsoft.ML.Fairlearn.reductions
```

Comment:
Did you mean to have the namespace be `reductions` with a lowercase r? Would be better to be a capital.

Reply:
@LittleLittleCloud looks like its still lowecase r. Is there a reason for it to be that way?

Reply:
Nope there's no reason for that but a mistake. I thought I fixed it but maybe not. Let me check